portfolio.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <style></style>
+  </head>
+  <body>
+    <header class="topHeader">
+      <p class="logo">TD</p>
+      <nav class="menu">
+        <a href="#" class="toggleNav">
+          <div class="menu_line"></div>
+          <div class="menu_line"></div>
+          <div class="menu_line"></div>
+        </a>
+        <ul class="menu_items"></ul>
```

Comment:
[OPTIONAL]Great job implementing the toggle button👍 But, I think you can consider using an icon or image for your toggle button, this will keep your code dry and simple.😊